### PR TITLE
[ASM] - Expander - Update ASM fields (4821)

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[EXPANDR-4821](https://jira-dc.paloaltonetworks.com/browse/EXPANDR-4821)

## Description

Adding comments to incident fields and updating descriptions.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by johnnywilkes — COMMENTED** [review]
@BigEasyJ , looks good.  Do we also want to include if the gridfield key is mandatory or optional such as: https://github.com/demisto/content/blob/56e2cd6df33f4a456d8fd0d0c72e21775288c836/Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Dev_Check_Details.json#L19 ?

**Comment by BigEasyJ** [comment]
@johnnywilkes Yah I would like to, but I would like a meeting to help with identifying and going over each one.

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Private_IP.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
             "script": "",
             "selectValues": null,
             "type": "shortText",
-            "width": 150
+            "width": 150,
+            "_comment": "The IP address of the relevant asset."
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Private_IP.json:19]:
```suggestion
            "_comment": "Mandatory: The IP address of the relevant asset."
```
to make the same as others

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Private_IP.json [hunk 2]
```diff
@@ -28,7 +29,8 @@
             "script": "",
             "selectValues": null,
             "type": "shortText",
-            "width": 150
+            "width": 150,
+            "_comment": "The source of the IP address."
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Private_IP.json:33]:
```suggestion
            "_comment": "Mandatory: The source of the IP address."
```

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Remediation.json [hunk 3]
```diff
@@ -15,7 +15,8 @@
             "script": "",
             "selectValues": null,
             "type": "shortText",
-            "width": 150
+            "width": 150,
+            "_comment": "Description of the remediation action."
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Remediation.json:19]:
```suggestion
            "_comment": "Mandatory: Description of the remediation action."
```

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Remediation.json [hunk 4]
```diff
@@ -41,7 +43,8 @@
             "script": "",
             "selectValues": null,
             "type": "shortText",
-            "width": 150
+            "width": 150,
+            "_comment": "Outcome of the remediation action."
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Remediation.json:47]:
```suggestion
            "_comment": "Mandatory: Outcome of the remediation action."
```

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Remediation.json [hunk 5]
```diff
@@ -54,7 +57,8 @@
             "script": "",
             "selectValues": null,
             "type": "shortText",
-            "width": 150
+            "width": 150,
+            "_comment": "timestamp of the remediation outcome."
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Remediation.json:61]:
```suggestion
            "_comment": "Mandatory: Timestamp of the remediation outcome."
```

File: Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Tags.json [hunk 6]
```diff
@@ -41,7 +43,8 @@
             "script": "",
             "selectValues": null,
             "type": "shortText",
-            "width": 150
+            "width": 150,
+            "_comment": "The source of tag. Ex: ServiceNOW, AWS, GCP, Tenable, etc."
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Tags.json:47]:
```suggestion
            "_comment": "Mandatory: The source of tag. Ex: ServiceNOW, AWS, GCP, Tenable, etc."
```

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
@BigEasyJ , just a few more to change

**Review by johnnywilkes — APPROVED** [review]
@BigEasyJ , looks good, ship it!

**Comment by ostolero** [comment]
Hi @BigEasyJ there seems to be a conflict with our master which blocks merge. What you can do is either solve the conflict yourself (it's simply the rn overlapping with other changes, shouldn't be too complicated). Or grant me permission to push to your branch, An explanation on how to do that is [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository).

**Comment by BigEasyJ** [comment]
@ostolero you are not a member of `PaloAltoNetworks` org for me to add you, you would need to put in a ticket with IT for access. I will push up the changes to fix the release versions.

**Review by ostolero — APPROVED** [review]
Looks great @BigEasyJ ! merging

**Pull request merged**